src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1197,6 +1197,8 @@ private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue cons
                 case SyntaxKind.BaseConstructorInitializer:
                 case SyntaxKind.ThisConstructorInitializer:
                     return new SourceLocation(((ConstructorInitializerSyntax)syntax).ArgumentList.OpenParenToken);
+                case SyntaxKind.PrimaryConstructorBaseType:
```

Comment:
>SyntaxKind [](start = 21, length = 10)

Is this tested?

Reply:
> Is this tested?

The need to test this code path is recorder in the test plan issue.

---
In reply to: [443927180](https://github.com/dotnet/roslyn/pull/45351#discussion_r443927180) [](ancestors = 443927180)

Reply:
Can you please link to the test plan issue?

---
In reply to: [443928479](https://github.com/dotnet/roslyn/pull/45351#discussion_r443928479) [](ancestors = 443928479,443927180)

Reply:
> Can you please link to the test plan issue?

[https://github.com/dotnet/roslyn/issues/40726](https://github.com/dotnet/roslyn/issues/40726), several items, right above "Productivity" section.

---
In reply to: [443929342](https://github.com/dotnet/roslyn/pull/45351#discussion_r443929342) [](ancestors = 443929342,443928479,443927180)